HPXMLtoOpenStudio/resources/hpxml_schematron/EPvalidator.xml

```diff
@@ -2276,6 +2282,8 @@
       <sch:assert role='ERROR' test='count(h:extension/h:WeekdayScheduleFractions) &lt;= 1'>Expected 0 or 1 element(s) for xpath: extension/WeekdayScheduleFractions</sch:assert>
       <sch:assert role='ERROR' test='count(h:extension/h:WeekendScheduleFractions) &lt;= 1'>Expected 0 or 1 element(s) for xpath: extension/WeekendScheduleFractions</sch:assert>
       <sch:assert role='ERROR' test='count(h:extension/h:MonthlyScheduleMultipliers) &lt;= 1'>Expected 0 or 1 element(s) for xpath: extension/MonthlyScheduleMultipliers</sch:assert>
+      <sch:assert role='ERROR' test='count(h:extension/h:ConstantCoefficients) &lt;= 1'>Expected 0 or 1 element(s) for xpath: extension/ConstantCoefficients</sch:assert>
+      <sch:assert role='ERROR' test='count(h:extension/h:TemperatureCoefficients) &lt;= 1'>Expected 0 or 1 element(s) for xpath: extension/TemperatureCoefficients</sch:assert>
```

Comment:
I don't love this. If the user enters all five values, some will simply be ignored and the user may not realize it. We'd like to restrict the inputs so they can't have one or more inputs from each fraction/coefficient set.

Maybe we just need one more assert like so:
`<sch:assert role='ERROR' test='count(h:extension/h:WeekdayScheduleFractions | h:extension/h:WeekendScheduleFractions | h:extension/h:MonthlyScheduleMultipliers) + count(h:extension/h:ConstantCoefficients | h:extension/h:TemperatureCoefficients) &lt;= 1'>Expected ......</sch:assert>`

I'm not sure if that works correctly, but there should be a way to do something like that.

Reply:
If you can figure out the assert here, we should add a test around it in test_validation.rb.

Reply:
It doesn't appear to be working correctly. When I enter the assert in EPvalidator.xml, the existing validation test fails on `base-misc-loads-large-uncommon.xml` where refrigerator has the usual schedule fractions/multipliers.

However, I wrote the new test. So hopefully we can figure out the correct assert here.

Reply:
I think I have it figured out. Testing now...